dir.props

```diff
@@ -26,12 +26,8 @@
     <IsDotNetFrameworkProductAssembly>true</IsDotNetFrameworkProductAssembly>
   </PropertyGroup>
   
-  <!--
-    Switching to the .NET Core version of the BuildTools tasks seems to break numerous scenarios, such as VS intellisense and resource designer
-    as well as runnning the build on mono. Until we can get these sorted out we will continue using the .NET 4.5 version of the tasks.
-  -->
   <PropertyGroup>
-    <BuildToolsTargets45>true</BuildToolsTargets45>
+    <BuildToolsTargets45 Condition="'$(OsEnvironment)'=='Windows_NT'">true</BuildToolsTargets45>
```

Comment:
Resources are broken with or without this condition. I left it because I'm not sure if the cloud test tasks built for .Net Core can run in windows builds. If Windows builds(both internal and external builds) can consume .Net Core version of the task libraries, then perhaps we should just remove the net45 folder from buildtools completely.